api/v1/status/atlasnetworkcontainer.go

```diff
@@ -0,0 +1,52 @@
+package status
+
+import "github.com/mongodb/mongodb-atlas-kubernetes/v2/api"
+
+// AtlasNetworkContainerStatus is a status for the AtlasNetworkContainer Custom resource.
+// Not the one included in the AtlasProject
+type AtlasNetworkContainerStatus struct {
+	api.Common `json:",inline"`
+
+	// ID record the identifier of the container in Atlas
+	ID string `json:"id,omitempty"`
+
+	// Provisioned is true when clusters have been deployed to the container
+	Provisioned bool `json:"provisioned,omitempty"`
```

Comment:
We could also remove this, as this is informative but might be stale and it is not actionable, instead the information in the peering status is what is used.